src/libcore/cmp.rs

```diff
@@ -462,6 +462,40 @@ impl<T: Ord> Ord for Reverse<T> {
     }
 }
 
+/// A wrapper newtype for providing a total order on a type that's normally partial.
```

Comment:
The brief summary seems to indicate that only `T: PartialOrd`s are accepted but the signature does not require that.

Reply:
That's completely true, but note that [`cmp::Reverse`](https://doc.rust-lang.org/std/cmp/struct.Reverse.html) also doesn't bound itself.

Reply:
Indeed, do you know why that is? Appears to be an oversight (what use is reverse if `T` does not implement `PartialOrd` ?)